CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 
 # Unreleased
 
+- [`Breaking`] Removes private key from the Account class to support keyless and passkey
```

Comment:
do we really need this change part of this PR? why not having it as part of the keyless PR?

Reply:
Yes, multikey accounts don't have a private key

Reply:
yes, [MultiKey](https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/core/crypto/multiKey.ts#L27) does not have a private key.

This change however is to remove the private key from the Account to support keyless and passkey - consider it is a breaking change, could we include it on the keyless PR instead of this PR?

Reply:
I think it makes sense to have it in this PR imo.  the [MultiKey](https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/core/crypto/multiKey.ts#L27) abstraction is justification enough i feel.

either way, the keyless stuff will be in the same cut.

Reply:
what "same cut"? the sdk doesnt follow aptos framework cut timeline. If we merge it in, next SDK release will include this change and would probably need to have a major version upgrade. 

If this change is to support keyless, we should get it in with the keyless integration PR